operations/node_management/Wipe_NCN_Disks.md

```diff
@@ -56,12 +55,23 @@ executed on **any type of management node** (master, storage, or worker).
 
     The `wipefs` command may fail if no labeled disks are found, which is an indication of a larger problem.
 
+1. (`ncn#`) On all NCNs, remove the volume groups.
+
+   > ***NOTE*** The Ceph volume group will only exist on storage-nodes, but this code snippet will work on all NCNs.
+
+    ```bash
+    local ceph_vgs='vg_name=~ceph*'
+    local metal_vgs='vg_name=~metal*'
+    for volume_group in $doomed_ceph_vgs $doomed_metal_vgs; do
+        vgremove -f -v --select $volume_group -y >/dev/null 2>&1
+    done
+    ```
+
 ## Advanced Wipe
 
-**This section is specific to utility storage nodes**. An advanced wipe includes stopping Ceph,
-deleting the Ceph volumes, and then wiping the disks and RAIDs.
+An advanced wipe includes handling storage-node specific items before running the [basic wipe](#basic-wipe).
```

Comment:
```suggestion
An advanced wipe includes handling storage-node-specific items before running the [basic wipe](#basic-wipe).
```

Reply:
storage-node does not need to be hyphenated here